<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Antenne</class>
 <widget class="QFrame" name="Antenne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Frame</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget_81">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_radio1">
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="verticalSpacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_adresse">
      <property name="text">
       <string>Adresse IP :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="adresse">
      <property name="inputMask">
       <string notr="true">000.000.000.000</string>
      </property>
      <property name="text">
       <string notr="true">127.0.0.1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_port">
      <property name="text">
       <string>Port :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="port">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>12000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="donneesTransmises">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Données transmises</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_83">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>176</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_topoSatRadio">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_hauteurSatRadio">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Hauteur :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_azimutSatRadio">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string extracomment="Azimuth (from the North)">Azimut (N) :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_rangeRateSatRadio">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Variation distance :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="hauteurSatRadio">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">+00° 00' 00&quot;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="azimutSatRadio">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">000° 00' 00&quot;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="rangeRateRadio">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Double-cliquer pour changer d'unités</string>
       </property>
       <property name="text">
        <string notr="true">+28000 km/h</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="indent">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frameFrequences">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>561</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="formLayoutWidget_84">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_frequenceMontante">
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_frequenceMontante">
       <property name="text">
        <string>Fréquence montante :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="frequenceMontante"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_frequenceMontanteReelle">
       <property name="text">
        <string>Fréquence réelle :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="frequenceMontanteReelle">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_dopplerMontant">
       <property name="text">
        <string>Doppler :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="dopplerMontant">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbl_baliseMontant">
       <property name="text">
        <string>Balise :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="baliseMontant">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lbl_modeMontant">
       <property name="text">
        <string>Mode :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="modeMontant">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lbl_signalAppelMontant">
       <property name="text">
        <string>Signal d'appel :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="signalAppelMontant">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_attenuationMontant">
       <property name="text">
        <string>Atténuation :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="attenuationMontant">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbl_delaiMontant">
       <property name="text">
        <string>Délai :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="delaiMontant">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_85">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>271</width>
      <height>162</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_frequenceDescendante">
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_frequenceDescendante">
       <property name="text">
        <string>Fréquence descendante :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="frequenceDescendante"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_frequenceDescendanteReelle">
       <property name="text">
        <string>Fréquence réelle :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="frequenceDescendanteReelle">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_dopplerDescendant">
       <property name="text">
        <string>Doppler :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="dopplerDescendant">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_attenuationDescendant">
       <property name="text">
        <string>Atténuation :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="attenuationDescendant">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbl_delaiDescendant">
       <property name="text">
        <string>Délai :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="delaiDescendant">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbl_baliseDescendant">
       <property name="text">
        <string>Balise :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="baliseDescendant">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lbl_modeDescendant">
       <property name="text">
        <string>Mode :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="modeDescendant">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lbl_signalAppelDescendant">
       <property name="text">
        <string>Signal d'appel :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="signalAppelDescendant">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="connexion">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Connecter</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="parametrageDefautRadio">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>Paramétrage par défaut</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ouvrirCatRotator">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ouvrir CatRotator</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameSatellite">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>63</y>
     <width>189</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="prochainAOS">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>21</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Prochain AOS dans </string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_82">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_nomsatRadio">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_nomsatRadio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="nomsatRadio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">ISS</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
